tests/e2e/helpers/keys.ts

```diff
@@ -0,0 +1,23 @@
+import { KeyTypesTexts } from './constants';
+
+export function getRandomKeyName(keyNameLength: number): string {
+    let result = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    const charactersLength = characters.length;
+    for (let i = 0; i < keyNameLength; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
```

Comment:
IMO You can use with [chance](https://www.npmjs.com/package/chance) instead of this

Reply:
Great library, thanks! Fix test with 'chance'